MTA_limit_trends/Comp/run_comp_grad_data_update.py

```diff
@@ -122,17 +120,29 @@
                  'htftegrd5', 'htftegrd6', 'htftegrd7', 'htftegrd8', \
                  'htftegrd9', 'htftegrd10', 'htftegrd11', 'htftegrd12']
 #
-#--- set them in the lists
-#
-group_name  = ['Compgradkodak', 'Compacispwr', 'Compsimoffset',\
-               'Gradablk', 'Gradahet', 'Gradaincyl', 'Gradcap', 'Gradfap', \
-               'Gradfblk', 'Gradhcone', 'Gradhhflex', 'Gradhpflex', 'Gradhstrut',\
-               'Gradocyl', 'Gradpcolb', 'Gradperi', 'Gradsstrut', 'Gradtfte']
-
-g_msid_list = [compgradkodak,    compacispwr,   compsimoffset, \
-               gradablk, gradahet, gradaincyl, gradcap, gradfap, \
-               gradfblk, gradhcone, gradhhflex, gradhpflex, gradhstrut,\
-               gradocyl, gradpcolb, gradperi, gradsstrut, gradtfte]
+#--- set them in the dictionary
+#
+
+GROUP_MSID_DICT = {
+    'Compgradkodak': compgradkodak,
+    'Compacispwr': compacispwr,
+    'Compsimoffset': compsimoffset,
+    'Gradablk': gradablk,
+    'Gradahet': gradahet,
+    'Gradaincyl': gradaincyl,
+    'Gradcap': gradcap,
+    'Gradfap': gradfap,
+    'Gradfblk': gradfblk,
+    'Gradhcone': gradhcone,
+    'Gradhhflex': gradhhflex,
+    'Gradhpflex': gradhpflex,
+    'Gradhstrut': gradhstrut,
+    'Gradocyl': gradocyl,
+    'Gradpcolb': gradpcolb,
+    'Gradperi': gradperi,
+    'Gradsstrut': gradsstrut,
+    'Gradtfte': gradtfte,
+}
```

Comment:
Structuring the grouping of category and lists of pseudo-msid's into a dictionary makes testing changes easier.